dm/pkg/conn/basedb.go

```diff
@@ -69,8 +69,8 @@ func (d *DefaultDBProviderImpl) Apply(config *config.DBConfig) (*BaseDB, error)
 		if loadErr := config.Security.LoadTLSContent(); loadErr != nil {
 			return nil, terror.ErrCtlLoadTLSCfg.Delegate(loadErr)
 		}
-		tlsConfig, err := util.ToTLSConfigWithVerifyByRawbytes(config.Security.SSLCABytes,
-			config.Security.SSLCertBytes, config.Security.SSLKEYBytes, config.Security.CertAllowedCN)
+		tlsConfig, err := util.ToTLSConfigWithVerify(config.Security.SSLCA,
```

Comment:
if we create task from dm api, there's no ca file, only ca bytes

/cc @lance6716

Reply:
Can we just convert these bytes to string key?

Reply:
https://github.com/pingcap/tiflow/pull/6913

this is the new API